Upstream-status: Backport

Backported from https://crrev.com/c/2228649

Signed-off-by: Maksim Sisov <msisov@igalia.com>
---
From 26eaf07d91c7c2d3e15a3ed74d4aaa2fb2bc46bc Mon Sep 17 00:00:00 2001
From: Maksim Sisov <msisov@igalia.com>
Date: Thu, 4 Jun 2020 10:25:20 +0000
Subject: [PATCH] Remove X11 deps from accessibility

This fixes pure ozone/wayland build on systems where
X11 is not available

Change-Id: Ib4e41c2fd149b48586623183d8fa656591415361
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2228649
Reviewed-by: Thomas Anderson <thomasanderson@chromium.org>
Commit-Queue: Maksim Sisov <msisov@igalia.com>
Cr-Commit-Position: refs/heads/master@{#775056}
---
 content/browser/BUILD.gn | 2 --
 1 file changed, 2 deletions(-)

diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 54b133ef11ff..91e33f8448e3 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -2338,8 +2338,6 @@ jumbo_source_set("browser") {
       "//build/config/linux/atspi2",
     ]
 
-    deps += [ "//ui/gfx/x" ]
-
     if (use_glib) {
       configs += [ "//build/config/linux:glib" ]
     }
-- 
2.25.1

